resources/js/components/ui/switch.tsx

```
@@ -36,21 +36,21 @@ export function Switch({ children, className, ...props }: SwitchProps) {
-                "dark:group-disabled:bg-muted-fg/30 dark:group-disabled:group-selected:inset-ring-muted-fg/30 dark:group-disabled:group-selected:bg-(--switch-bg)",
+                "dark:group-disabled:bg-muted-foreground/30 dark:group-disabled:group-selected:inset-ring-muted-foreground/30 dark:group-disabled:group-selected:bg-(--switch-bg)",
             )}
           >
             <span
               aria-hidden="true"
               className={twJoin(
-                "pointer-events-none relative inline-block size-4.5 translate-x-0 rounded-full border border-transparent bg-white shadow-sm ring ring-fg/5 transition duration-200 ease-in-out sm:size-3.5",
+                "pointer-events-none relative inline-block size-4.5 translate-x-0 rounded-full border border-transparent bg-white shadow-sm ring ring-foreground/5 transition duration-200 ease-in-out sm:size-3.5",
                 values.isSelected &&
-                  "rtl:-translate-x-4 rtl:sm:-translate-x-3 bg-(--switch) shadow-(--switch-shadow) ring-(--switch-ring) group-disabled:shadow-sm group-disabled:ring-secondary-fg/5 ltr:translate-x-4 ltr:sm:translate-x-3",
+                  "rtl:-translate-x-4 rtl:sm:-translate-x-3 bg-(--switch) shadow-(--switch-shadow) ring-(--switch-ring) group-disabled:shadow-sm group-disabled:ring-secondary-foreground/5 ltr:translate-x-4 ltr:sm:translate-x-3",
```

Biome (2.4.13)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)